.github/workflows/ci.yml

```diff
@@ -64,6 +64,10 @@ jobs:
       if: matrix.os == 'ubuntu'
       run: sudo apt-get install -y ragel
 
+    - name: Install ragel
+      if: matrix.os == 'macos'
+      run: brew install ragel
+
```

Comment:
Ragel is used to generate the `char_ref.c` file which should already be in the repo. There's a newer version of Ragel that is not backwards compatible. Indeed, when I try to generate `char_ref.c` with the newer version, it fails.

I'm not sure what version brew is going to install. Given that `char_ref.c` only needs to change if new entities are added, it's probably not worth testing.

Reply:
OK, how would you prefer me to proceed? We could remove the package test from macos; or remove the package test altogether. But since this is green right now, maybe we just let it go for now?

Reply:
Perhaps I'm confused, but I don't think the package test is using ragel at all. When I look at the build log for the macos package test, it isn't invoking it, it's just compiling the existing `char_ref.c` in the repo. Were you getting test failures without that step?

Reply:
Also, I apologize for the delay in responding. I'm getting buried by a deluge of GitHub emails. Every time one of my students starts an assignment using GitHub classroom, GitHub feels the need to email me!

Reply:
This commit was added because CI started failing due to ragel missing on the macos container. Here's a deep link to the failure: https://github.com/rubys/nokogumbo/runs/2112263537?check_suite_focus=true

Reply:
Huh, that's really surprising. The checks on _this_ commit didn't use it. The only thing I can imagine is the time stamps being wrong on the checkout in the failure. Probably not worth tracking down. And it looks like brew is using the stable version of ragel anyway so I guess there's no harm.